source/Logger.cpp

```diff
@@ -370,6 +372,8 @@ void FPSciLogger::logTargetTypes(const Array<shared_ptr<TargetConfig>>& targets)
 			"'" + config->destSpace + "'",
 			String(std::to_string(config->size[0])),
 			String(std::to_string(config->size[1])),
+			String(std::to_string(config->symmetricEccH)),
```

Comment:
This results in a string value of "0" or "1" being written to the file instead of "true" or "false". You can see examples of how we setup these strings in `updateSessionEntry()` or just above this in `logTargetTypes()` (for the `jumpEnabled` value). I'd suggest moving to this style of string reporting here.

Having said that we are inconsistent in our use of capitalization in these "true" and "false" fields, so maybe we should create an issue to harmonize this for the entire logger class.

Reply:
I agree it's a problem. I'll add a check box to the big database restructuring PR #260  and we can roll the fix in there.

Reply:
Fixed!